ostd/src/boot/kcmdline.rs

```diff
@@ -40,6 +42,7 @@ pub enum ModuleArg {
 #[derive(Debug)]
 pub struct KCmdlineArg {
     initproc: InitprocArgs,
+    log_level: Option<Level>,
     module_args: BTreeMap<String, Vec<ModuleArg>>,
```

Comment:
Oh, I forgot to remove the `log_level` field since it is no longer used after using `ostd.log_level` as the kernel command line option. I will do it in the next version.

Reply:
`InitprocArgs` is also useless for ostd itself, right?

Reply:
Yes, `InitprocArgs` is only used in `aster-nix` to spawn the first user process.

Reply:
Could you simplify `kcmdline` module by moving as many code as possible to `aster-nix`? This is probably best done in another PR.